src/components/Quiz/Quiz.tsx

```
@@ -0,0 +1,511 @@
+import {
+  CaretLeft,
+  CaretRight,
+  Clock,
+  Palette,
```

ESLint

[error] 'Palette' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)